bigtop-manager-dao/src/main/java/org/apache/bigtop/manager/dao/enums/DBType.java

```diff
@@ -26,6 +26,7 @@
 @Getter
 public enum DBType {
     MYSQL("mysql", "MYSQL"),
```

Comment:
How about we keep the desc the same as productName, `MySQL` here for example?

Reply:
changed to MySQL